docs/install.md

```diff
@@ -25,6 +25,33 @@ ensure that your Firedrake environment is activated:
 
 Furthermore, the `gadopt` Python module should be available for you to use.
 
+## Using the G-ADOPT library on Gadi
+
+The G-ADOPT team maintains an up-to-date Firedrake and G-ADOPT installation 
+on NCI's Gadi HPC system. To use G-ADOPT on Gadi, create an NCI account, then follow [this link](https://my.nci.org.au/mancini/project/fp50/join) to request to join the fp50 project. Once your request is approved, you can
+access G-ADOPT on Gadi by running:
+
+    module use /g/data/fp50/modules
+    module load firedrake
+
+To use G-ADOPT in you batch scripts, you'll need to add the following to your
+PBS flags:
+
+    -lstorage=gdata/fp50
+
+By default, the latest release of G-ADOPT will be available when you load
+the firedrake module. If you wish to use your own G-ADOPT installation, set
+the environment variable `MY_GADOPT` before loading the firedrake module.
+
+    export MY_GADOPT=/g/data/ab12/path/to/my/gadopt
+    module use /g/data/fp50/modules
+    module load firedrake
+
+You can also use G-ADOPT in a JupyterLab session on the [ARE](https://opus.nci.org.au/spaces/Help/pages/162431120/ARE+User+Guide) at NCI. When launching a
```

Comment:
Maybe obvious but you also need to request storage for the fp50 project not just the modules in the advanced options section. Is it worth writing this too?